gnome-extra/gnome-commander/gnome-commander-1.14.0.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+GNOME2_LA_PUNT="yes"
+
+inherit gnome2 optfeature
+
+DESCRIPTION="A graphical, full featured, twin-panel file manager"
+HOMEPAGE="https://gcmd.github.io/"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="chm exif gsf pdf samba taglib test +unique"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	app-text/yelp-tools
+	dev-libs/atk
+	dev-libs/glib:2
+	gnome-base/gconf:2
+	media-libs/fontconfig
+	media-libs/freetype
+	media-libs/harfbuzz:=
+	x11-libs/cairo
+	x11-libs/gdk-pixbuf:2
+	x11-libs/gtk+:2
+	x11-libs/pango
+	chm? ( dev-libs/chmlib )
+	exif? ( >=media-gfx/exiv2-0.14 )
+	gsf? ( gnome-extra/libgsf:= )
+	pdf? ( >=app-text/poppler-0.18 )
+	samba? ( gnome-base/gvfs[samba] )
+	taglib? ( >=media-libs/taglib-1.4 )
+	unique? ( >=dev-libs/libunique-0.9.3:1 )
+"
+BDEPEND="
+	dev-util/gtk-doc-am
+	sys-devel/gettext
```

Comment:
Should be `>=sys-devel/gettext-0.19.7`

Reply:
only available in portage is 0.21, do you really think the explicit version is needed?

Reply:
In general we keep minimum version reqs until after some time (2 years) they have left the tree. gnome proj may have different standards.

Reply:
Yeah, I don't really have a preference. I guess it just seems easier to put the version in so you don't have to think about it. :shrug:

Reply:
Updated, along with the addition of `sys-devel/flex`